src/js/tabs/configuration.js

```diff
@@ -1235,7 +1235,17 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             PID_ADVANCED_CONFIG.use_unsyncedPwm = $('input[id="unsyncedPWMSwitch"]').is(':checked') ? 1 : 0;
             PID_ADVANCED_CONFIG.motor_pwm_rate = parseInt($('input[name="unsyncedpwmfreq"]').val());
             PID_ADVANCED_CONFIG.gyro_sync_denom = parseInt(gyroSelectElement.val());
-            PID_ADVANCED_CONFIG.pid_process_denom = parseInt(pidSelectElement.val());
+
+            const value = parseInt(pidSelectElement.val());
+
+            let newValue = undefined;
```

Comment:
Maybe is better to give the variables a name more descriptive, this are too generic.

Reply:
One thing I want to do for 10.8 is to refactor and centralise these analytics helper functions, now that we have well established patterns.